pkg/apis/componentconfig/types.go

```diff
@@ -54,6 +54,13 @@ type ControllerManagerConfiguration struct {
 	// ServiceCatalogKubeconfigPath is the path to the kubeconfig file with
 	// information about the service catalog API server.
 	ServiceCatalogKubeconfigPath string
+	// InsecureSkipVerify controls whether a client verifies the
```

Comment:
This comment was shamelessly stolen from [crypto/tls](https://github.com/golang/go/blob/master/src/crypto/tls/common.go#L442)